### DISCUSSION TITLE
Security certificates

### DISCUSSION THREAD

**Discussion by MadisJarve** [rodlie/cyan#114] [General]
Hi!

I work in a big enterprise and some people are interested in using this software. Do you perhaps hold any security certificates for this software? (Like ISO27001 or SOC2). Please let me know if you do, it would make it a lot easier to start using it.

**Comment by rodlie** [comment]
Hi,

There are no certificates for the application.

Cyan bundle several open source components (Qt and several image formats etc), some are currently outdated in the current binary release (1.2.4). I can update the components used to latest stable and do a new binary release of Cyan (1.2.5).

If you want to contact me privately use ole.andre.rodlie@gmail.com.
